monitoring/kaggle/scripts/test-kaggle-integration.sh

```
@@ -0,0 +1,74 @@
+
+# Check environment variables
+if [ -z "$KAGGLE_USERNAME" ] || [ -z "$KAGGLE_API_KEY" ]; then
+    echo "${RED}❌ KAGGLE_USERNAME or KAGGLE_API_KEY not set${NC}"
+    exit 1
+fi
+
+echo "✅ Environment variables loaded"
+
+# Test Kaggle API directly
+echo "🧪 Testing Kaggle API connection..."
+HTTP_STATUS=$(curl -s -o /dev/null -w "%{http_code}" -u "$KAGGLE_USERNAME:$KAGGLE_API_KEY" https://www.kaggle.com/api/v1/datasets/list?page=1&pageSize=1)
```

Shellcheck (0.11.0)

[warning] This & terminates the command. Escape it or add space after & to silence.

(SC1132)